docs/source/user-guide/configuration/use-condarc.rst

```diff
@@ -162,7 +162,8 @@ The precedence by which the conda configuration is built out is shown below.
 Each new arrow takes precedence over the ones before it. For example, config
 files (by parse order) will be superseded by any of the other configuration
 options. Configuration environment variables (formatted like ``CONDA_<CONFIG NAME>``)
-will always take precedence over the other 3.
+will always take precedence over all config files. And command line parameters
```

Comment:
```suggestion
will always take precedence over all config files. Command line parameters
```
Let's not start a sentence with "and". If we want to emphasize this sentence, we could italicize it.